src/shared/sam/cli/samCliVersionValidator.ts

```diff
@@ -129,17 +125,17 @@ export class SamCliVersionValidator {
             case SamCliVersionValidation.VersionTooLow:
             case SamCliVersionValidation.VersionNotParseable:
             default:
-                actions.push(SamCliVersionValidator.ACTION_GO_TO_SAM_CLI_PAGE)
+                actions.push(DefaultSamCliVersionValidator.ACTION_GO_TO_SAM_CLI_PAGE)
                 break
         }
 
         return actions
     }
 
     private async handleUserResponse(userResponse: string): Promise<void> {
-        if (userResponse === SamCliVersionValidator.ACTION_GO_TO_SAM_CLI_PAGE) {
-            await vscode.commands.executeCommand('vscode.open', vscode.Uri.parse(samAboutInstallUrl))
-        } else if (userResponse === SamCliVersionValidator.ACTION_GO_TO_AWS_TOOLKIT_PAGE) {
+        if (userResponse === DefaultSamCliVersionValidator.ACTION_GO_TO_SAM_CLI_PAGE) {
+            await vscode.env.openExternal(vscode.Uri.parse(samAboutInstallUrl))
+        } else if (userResponse === DefaultSamCliVersionValidator.ACTION_GO_TO_AWS_TOOLKIT_PAGE) {
             // TODO : direct to AWS Toolkit page when there is one
```

Comment:
I did write an issue for this (https://github.com/aws/aws-toolkit-vscode/issues/400); thinking we can punt for now since that message shouldn't ever pop up. At the very least, we don't pop up a button (just a nebulous error) so in a worst-case scenario, we don't put up a broken error box.

![overversion](https://user-images.githubusercontent.com/29374703/55109905-15a49400-5094-11e9-815e-68683d48bc2e.PNG)